man/fi_mr.3.md

```diff
@@ -46,6 +50,11 @@ void * fi_mr_desc(struct fid_mr *mr);
 
 uint64_t fi_mr_key(struct fid_mr *mr);
 
+int fi_mr_raw_key(struct fid_mr *mr, uint8_t *raw, size_t size);
+
+int fi_mr_map_key(struct fid_domain *domain, uint8_t *raw, size_t size,
+    uint64_t *key);
```

Comment:
Will peer need to 'unmap' or free the key, so that the peer provider can release any allocated memory?